### PR TITLE
Command for doing a basic check of a configuration file.

### PR DESCRIPTION
**Pull request by inercia**
## What does this PR change?

Add a new command to the `kubic-init` binary for performing a very simple check for a config file (it just tries to load it).
Check the sample configuration files included in the repo as part of the CI jobs.

## Documentation

- No documentation needed

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,6 +106,9 @@ check: $(GO_BIN)/golint
 	terraform fmt
 lint: check
 
+check-config: $(KUBIC_INIT_EXE)
+	@for f in config/kubic-init.yaml* ; do $(KUBIC_INIT_EXE) checkconfig --config $$f ; done
```
Comment (MalloZup) [inline Makefile]: `config/*.yaml` ?

Comment (inercia) [reply Makefile]: I'm not sure there will be only `kubic-init` configuration in that directory. What if there was some yaml file for loading another CNI driver, for example? I would keep the glob that matches the things we know we can load: `kubic-init` `yaml` files.

Comment (MalloZup) [reply Makefile]: yop

Comment (nirmoy) [reply Makefile]: why not config/samples/*.yaml and put kubic-init.config.sample in there as well

Comment (MalloZup) [reply Makefile]: i'm open for both. feel free to chose one

**Pull request merged**